### PR TITLE
Hydroponics can now make sci points

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
## About The Pull Request

Blatant Ros/Tim buff
Makes each known to me seed packets be able for a 1 time DA use for a small amount of points
Adds a few more tips to tell players about it 

## Why It's Good For The Game

Gives some more of the useless plants a use, looking at space-thisle and corps-flower
Well not able to get to much points from round start vender (Around 2500~points), but still allows hydro and sci to interact for different types of seeds  

## Changelog
:cl:
add: Seed packs now have RnD points inside them
add: New tips that reflect the now use seed packs have for Rnd / Chems affect on plants
code: Made the research file look nice rather then an eye harming list 
/:cl:

### CODE REVIEW

File: strings/tips.txt [hunk 1]
```diff
@@ -148,6 +149,8 @@ As a Botanist, you can hack the MegaSeed Vendor to get access to more exotic see
 As a Botanist, you can mutate the plants growing in your hydroponics trays with unstable mutagen or, as an alternative, crude radioactives from chemistry to get special variations.
 As a Botanist, you should look into increasing the potency of your plants. This increases the size, amount of chemicals, points gained from grinding them in the biogenerator, and lets people know you are a proficient botanist.
 As a Botanist, you can combine production trait chemicals just like a Chemist. Chlorine (blumpkin) + radium and phosphorus (glowshrooms) equals unstable mutagen!
+As a Botanist, Rnd might pester you for different type of seed packets, perhaps a deal can be made in turn.
+As a Botanist, plants have different reactions to different chemicals, think logical like how in many gardeners use ash as a mulch.
```
Comment (Ghommie) [inline strings/tips.txt]: Not much of an useful tip and fairly ambiguous with chemical reactions too. Remove.

File: strings/tips.txt [hunk 2]
```diff
@@ -148,6 +149,8 @@ As a Botanist, you can hack the MegaSeed Vendor to get access to more exotic see
 As a Botanist, you can mutate the plants growing in your hydroponics trays with unstable mutagen or, as an alternative, crude radioactives from chemistry to get special variations.
 As a Botanist, you should look into increasing the potency of your plants. This increases the size, amount of chemicals, points gained from grinding them in the biogenerator, and lets people know you are a proficient botanist.
 As a Botanist, you can combine production trait chemicals just like a Chemist. Chlorine (blumpkin) + radium and phosphorus (glowshrooms) equals unstable mutagen!
+As a Botanist, Rnd might pester you for different type of seed packets, perhaps a deal can be made in turn.
```
Comment (Ghommie) [inline strings/tips.txt]: Remove, useless tip.

File: strings/tips.txt [hunk 3]
```diff
@@ -55,7 +55,8 @@ As a Scientist, you can disable anomalies by scanning them with an analyzer, the
 As a Scientist, researchable stock parts can seriously improve the efficiency and speed of machines around the station. In some cases, it can even unlock new functions.
 As a Scientist, you can generate research points by letting the tachyon-doppler array record increasingly large explosions.
 As a Scientist, getting drunk just enough will speed up research. Skol!
-As a Scientist, you can get points by placing slime cores into the deconstructive analyzer! This even works with crossbred slime cores. 
+As a Scientist, you can get points by placing slime cores into the deconstructive analyzer! This even works with crossbred slime cores.
+As a Scientist, work with botanists to get different types of seeds, as each type of seed can be used in the deconstructive analyzer for a small amount of Rnd type points!
```
Comment (Ghommie) [inline strings/tips.txt]: Merge the two tips together: `As a Scientist, research points can be generated by destroying specific objects such as slime cores, plant seeds and anomaly cores with the destructive analyzer!`

It's "destructive analyzer", not "deconstructive analyzer" by the by.

**Review by Ghommie — CHANGES_REQUESTED** [review]
In the long run, with new plants possibly being added, this is a cheapshot inefficient way to make seeds any good for science points making. It'd be more reliable to run a similar loop on the subsystem Initialize():
```
for(var/obj/item/seed/S in subtypesof(/obj/item/seed))
	var/list/L = list()
	L[TECHWEB_POINT_TYPE_GENERIC] = 50 + initial(S.rarity) * 2
	techweb_point_items[S] = L
```

Like, heck, the list is so big it occupies half the file now.

File: strings/tips.txt (hunk 1 above)
Comment (Trilbyspaceclone) [reply strings/tips.txt]: I dont want to spoon feed the answers like holy water heals plants.

File: strings/tips.txt (hunk 2 above)
Comment (Trilbyspaceclone) [reply strings/tips.txt]: How is it useless

File: strings/tips.txt (hunk 3 above)
Comment (Trilbyspaceclone) [reply strings/tips.txt]: Those are two wildly different tips

Comment (Ghommie) [reply strings/tips.txt]: No, both can be analyzed with the destructive analyzer and provide points, there is nothing wildly different, Trilby.

File: strings/tips.txt (hunk 2 above)
Comment (Ghommie) [reply strings/tips.txt]: It's useless, not a tip, just a memo of some unlikely roleplaying situation.

File: strings/tips.txt (hunk 1 above)
Comment (Ghommie) [reply strings/tips.txt]: Remove.

File: strings/tips.txt (hunk 3 above)
Comment (Trilbyspaceclone) [reply strings/tips.txt]: They are different tips, end of story

**Comment by davidchan13** [comment]
None of these are worth the effort to vend them, mutate them and harvest them. 10375 points assuming they go through and mutate every plant and miners retrieve every lavaland flora for them the seed. 

On top of the stats and traits of the seeds not even being considered in the point values.

**Comment by Trilbyspaceclone** [comment]
@davidchan13 Yes, why would rnd care about how the stats are of new plants?

**Comment by yourdoom9898** [comment]
@Trilbyspaceclone I mean, botany in part is the study of the genetics of the plant, which in spess is abstracted as a plant's stats, therefore any improvement over baseline stats would constitute research towards understanding that plant's genetic template, and how to interact with it to galactic benefit (and make a profit off it, this is a mega-corporation we work for after all)

**Comment by Trilbyspaceclone** [comment]
That would be too good even if it was +1 point per stat. Its ment to be a small thing not pwer gamed into a meme to get 800 per seed in vender

**Comment by davidchan13** [comment]
Honestly this would make more sense from IC and OOC standpoint to make it based off traits and stats over the actual base plant, or by analyzing traits or stats on plant disks and generating points if they are better or not already in the database.

**Comment by Trilbyspaceclone** [comment]
@davidchan13 Not from a game stand point.
It would be to powerfull

**Comment by davidchan13** [comment]
These are all one time benefits with a max benefit of 10k points. 20 minutes in Xenobotany produces more points.

**Comment by Trilbyspaceclone** [comment]
Xenobio takes way more then 20 mins to get 10k tho
And this again is not ment to be pwer gamed just a small way for botney to interact with sci

**Comment by Trilbyspaceclone** [comment]
I dont want this power creep

**Review by Ghommie — APPROVED** [review]

**Pull request merged**